colorbleed/plugins/fusion/create/create_saver.py

```diff
@@ -0,0 +1,46 @@
+import os
```

Comment:
Note for future selfs, change the module name: too generic

Reply:
Thanks - fixed.